creator/Documents/ScoreboardIntroduction.md

```diff
@@ -201,20 +200,18 @@ If you try to use this command with the "all players" or '@a' selector, only rea
 
 ### reset
 
-To remove a player from an objective (which removes them from the scoreboard), use:
+The **reset** operation removes a player from all objectives, resetting their value. The objective argument is optional, and will reset a player only in that objective.
```

Comment:
I felt this and line 210 were important to change. `/scoreboard players reset` without an objective will reset *all* of that player's scores, which can have unintended side effects.